source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,26 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
+	unlock: override func
+}
+
+MutexGlobal: class extends Mutex {
+	_globalMutex := static Mutex new(MutexType Safe)
+	init: func
+	lock: override func {
+		This _globalMutex lock()
+	}
+	unlock: override func {
+		This _globalMutex unlock()
+	}
+	cleanup: static func {
```

Comment:
I like `dispose` too, but it is already used in Closure to free the `instances` of Closures. Maybe we should use something different to clean up `instances` and `classes`.